### PR TITLE
VirusTotal Presence

### PR DESCRIPTION
**Pull request by MauriceX24**
Request
LkyVYRyO

### CODE REVIEW

File: VirusTotal/dist/presence.js [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+    presenceData.startTimestamp = browsingStamp;
+    video = document.querySelector("#app > div.fixed.fixed--top > div > div.fm-player__btn-container > button.btn.playbutton.fm-player__btn.fm-player__btn--stop.playbutton--playing");
+    if (document.querySelector("#app > div.fixed.fixed--top > div > div.fm-player__btn-container > button.btn.playbutton.fm-player__btn.fm-player__btn--stop.playbutton--playing").className == "btn playbutton fm-player__btn fm-player__btn--stop playbutton--play") {
+        
+        presence.details = "Its a Test toast";
+        presence.setActivity(presenceData);
+        
+    }
+    else if (document.location.pathname == "/") {
+        playingnow = document.querySelector("#app > div.fixed.fixed--top > div > a > div > div > span > b").innerText;
```
Comment (Bas950) [inline VirusTotal/dist/presence.js]: same with all of this

File: VirusTotal/dist/presence.js [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+   }
+       
+
+ 
+   
+        //presenceData.state = "Lurking on LautFM"; break;
+        //default : presenceData.state = "ZZzzzZZ"; 
+        ////
+        //playingnow = document.querySelector("#app > div.fixed.fixed--top > div > a > div > div > span > b").innerText;
+        //presenceData.details = "Playing " + playingnow;
```
Comment (Bas950) [inline VirusTotal/dist/presence.js]: Why is this in here?

File: VirusTotal/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        "name": "MauriceX",
+        "id": "268768664411439115"
+    },
+    "service": "VirusTotal",
+    "description": {
+        "en": "Check every Software for your safety",
+        "de": "Überprüfe deine Dateien für deine Sicherheit"
+    },
+    "url": [
+        "www.virustotal.com"],
```
Comment (Bas950) [inline VirusTotal/dist/metadata.json]: Just make it url: "www.virustotal.com"

**Comment by Bas950** [comment]
somehow all  the comments are backwords xD

File: VirusTotal/dist/presence.js (hunk 2 above)
Comment (MauriceX24) [reply VirusTotal/dist/presence.js]: It is a template if i forgot anything :D
should I remove this?

Comment (Bas950) [reply VirusTotal/dist/presence.js]: yea try to keep it as clean as possible, but you can leave notes etc. just not a whole template

**Pull request merged**